### PR TITLE
Upgrade simdjson to 3.8.0 (from 3.2.0)

### PR DESCRIPTION
**Pull request by mbasmanova**
Summary: [simdjson] third-party library import

Reviewed By: Yuhta

Differential Revision: D56031937

### CODE REVIEW

File: CMake/resolve_dependency_modules/README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ by Velox. See details on bundling below.
 | fbthrift          | v2024.04.01.00  | No       |
 | DuckDB (testing)  | 0.8.1           | Yes      |
 | cpr (testing)     | 1.10.15         | Yes      |
+| simdjson          | 3.8.0           | Yes      |
```
Comment (kgpai) [inline CMake/resolve_dependency_modules/README.md:42]: I think simdjson already exists in the list above.

**Comment by mbasmanova** [comment]
CC: @PHILO-HE

File: CMake/resolve_dependency_modules/README.md (hunk 1 above)
Comment (mbasmanova) [reply CMake/resolve_dependency_modules/README.md:42]: That's right. Let me fix.

**Review by Yuhta — APPROVED** [review]

**Review by assignUser — APPROVED** [review]

**Comment by PHILO-HE** [comment]
@mbasmanova, it is good to directly upgrade to this version. Thanks!

**Pull request closed without merging**